content/about.md

```diff
@@ -44,3 +44,4 @@ Mantra studied Mathematics at university and realised most of it won't be used a
 * [Marie Gronemeyer](https://github.com/muppi1993)
 * [Nicolas Dolan](https://github.com/Nicolas-Dolan)
 * [Alex Hill](https://github.com/hillalex)
+* [Lekan Anifowoshe](https://github.com/carbowizzy)
```

Comment:
Lekan actually updated his github id but didn't change it here !

```suggestion
* [Lekan Anifowoshe](https://github.com/LekanAnni)
```

Reply:
updated